frontend/src/Points.tsx

```diff
@@ -93,9 +94,26 @@ const Points = (props: IProps): JSX.Element => {
 
   thresholdStr += ` (next threshold: ${nextThreshold}分)`;
 
+  const { results: scoreTransitions } = explainScoring({
+    params: props.gameScoringParameters,
+    smaller_landlord_team_size: props.smallerTeamSize,
+    num_decks: props.numDecks,
+  });
+
   return (
     <div className="points">
       <h2>Points</h2>
+      <ProgressBar
+        checkpoints={scoreTransitions
+          .map((transition) => transition.point_threshold)
+          .filter(
```

Comment:
I want to call out this filter -- if there are checkpoint circles really close to the edges it makes the progress bar look bad. And AFAIK this will generally only happen at the 5-point threshold which is somewhat trivial.